ipaserver/plugins/join.py

```diff
@@ -85,7 +85,7 @@ class join(Command):
     has_output = tuple()
     use_output_validation = False
 
-    def execute(self, hostname, **kw):
+    def execute(self, hostname, **kwargs):
```

Comment:
There is nothing wrong with ``kw`` here.

Reply:
True that.